CHANGELOG.md

```diff
@@ -20,6 +17,12 @@ See [RELEASE](./RELEASE.md) for workflow instructions.
 
 * [#5807](https://github.com/spacemeshos/go-spacemesh/pull/5807) Implement SMIP-0002: remove vesting vault cliff.
 
+* [#5791](https://github.com/spacemeshos/go-spacemesh/pull/5791) Speed up ATX queries.
+  This also fixes ambiguity of nonces for equivocating identities.
+
+* [#5856](https://github.com/spacemeshos/go-spacemesh/pull/5856) Bump github.com/spacemeshos/api/release/go to v1.37.0.
+  Updated v2alpha1.ActivationTx type used in v2alpha1.Activation(Stream)Service.
```

Comment:
I don't think this needs to be included in the changelog - v2alpha1 is still subject to change and we never had it mentioned in the CHANGELOG before I believe